lnwallet/channel.go

```diff
@@ -4726,13 +4727,15 @@ func (lc *LightningChannel) ProcessChanSyncMsg(
 // view (settling unsettled HTLCs), commitment weight and feePerKw, after
 // applying the HTLCs to the latest commitment. The returned balances are the
 // balances *before* subtracting the commitment fee from the initiator's
-// balance.
+// balance. It accepts a "dry run" feerate argument to calculate a potential
+// commitment transaction fee.
 //
 // If the updateState boolean is set true, the add and remove heights of the
 // HTLCs will be set to the next commitment height.
 func (lc *LightningChannel) computeView(view *htlcView, remoteChain bool,
-	updateState bool) (lnwire.MilliSatoshi, lnwire.MilliSatoshi,
-	lntypes.WeightUnit, *htlcView, error) {
+	updateState bool, dryRunFee fn.Option[chainfee.SatPerKWeight]) (
+	lnwire.MilliSatoshi, lnwire.MilliSatoshi, lntypes.WeightUnit,
+	*htlcView, error) {
```

Comment:
I don't like this because every callsite would have to go through the update logs to find the "current" feerate. Option is ugly but don't see a better way to do this

Reply:
Got it. Maybe it'll have to wait til I clean this area up a bit more.